tests/resources/Vsphere-Util.robot

```diff
@@ -287,6 +287,15 @@ Check CVM Folder Path
     # If it is VC - we should find the vch in a folder named after the VCH.
     Run Keyword If  '%{HOST_TYPE}' == 'VC'  Should Contain  ${cvm-path}  vm/%{VCH-NAME}/${cvm-name}
 
+# Check VCH VM Folder Path will confirm that the VCH VM is in the correct place. if passed no arguments it will use the env var VCH-NAME for the check
+Check VCH VM Folder Path
```

Comment:
I'd prefer this be a generic _Check VM Folder Path_ and we pass the name -- we'll need to test appropriate Container placement as well...

Reply:
This already has an input to the function. But if it is called without the input then we will use the env var `VCH_NAME`. I wrote it this way since most of the tests in the CI use the same VCH, but there is a need for the input for multiVCH functions.

Reply:
I could combine the above Check and this one for the generic approach.

Reply:
making this change.